tgui/packages/tgui/interfaces/GeneTek/tabs/MutationsTab.js

```diff
@@ -39,11 +40,18 @@ export const MutationsTab = (props, context) => {
           onClick={() => setSortMode(sortMode === "time" ? "alpha" : "time")}>
           Sort Mode
         </Button>
+        <Button.Checkbox
+          inline
+          content="Show Sequence"
+          checked={showSequence}
+          onClick={() => showSequenceToggle(!showSequence)}
+        />
       </Section>
       {bioEffects.map(be => (
         <BioEffect
           key={be.ref}
-          gene={be} />
+          gene={be}
+          hideSequence={!showSequence} />
```

Comment:
All of this inversion is making my head spin!

No, but in all seriousness, can do:
```suggestion
          showSequence={showSequence} />
```
Then alter `BioEffect.js` accordingly (remove the `!` and rename the variable).

Reply:
Inversion was to get around leveraging default falsey for unknown.

Your suggestion will result in easier to read and the design intent will be explicit. 👍